doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
Well, it removes the roadblock of wondering whether something is ready....and shifts it to devs being willing to formally sign off.

IMO, the new formal PR review stuff would make it much easier to track who's signed off.